studio.ps1

```diff
@@ -67,10 +67,24 @@ function Stage-Three {
     $kitArgs = "/q"
 
     if ($KITTYPE -eq "HLK") {
-        if (Test-Path -Path "$PSScriptRoot\Kits\HLK${HLKKITVER}\HLKSetup.exe") {
-            $kitPath = "$PSScriptRoot\Kits\HLK${HLKKITVER}\HLKSetup.exe"
-        } else {
-            $kitPath = "$PSScriptRoot\Kits\HLK${HLKKITVER}Setup.exe"
+        $mountedDisks = Get-WmiObject -Class Win32_LogicalDisk | Where-Object { $_.DriveType -eq 5 }
+        $foundHCK = $false
+
+        foreach ($disk in $mountedDisks) {
+            $hckSetupPath = Join-Path -Path $disk.DeviceID -ChildPath "HLKSetup.exe"
+            Write-Output "Checking $hckSetupPath..."

```

Comment:
Please remove this. It outputs no log when e.g., testing `"$PSScriptRoot\Kits\HLK${HLKKITVER}\HLKSetup.exe"`, and I don't see anything special here.

Reply:
I expect to have a log there because we are looking in many places for the HLK installer and this log simplifies debugging.

Reply:
These logs don't tell much though; they only enumerate drive letters.